src/utils/shim.ts

```diff
@@ -0,0 +1,4 @@
+import atob from 'atob';
+
+// Shims for atob being undefined in node.js prior version 14
+if (!globalThis.atob) globalThis.atob = atob;
```

Comment:
@Ethella Looks good to me. Just dropped the test coverage too low 🤣

Reply:
Wasn't aware of tests 😭. Will make them up